modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {systemSettingsPageTest} from '../../fixtures/systemSettingsPageTest';
+
+export const test = mergeTests(loginTest(), systemSettingsPageTest);
+
+test('@LPD-26435 Close active menu', async ({page}) => {
```

Comment:
[nit] This is not very descriptive of what we are testing. We are not testing closing of active menu, it's just a step that happens in the test.

Reply:
Got it, changed!